### PR TITLE
[#11] working docker

### LINKED ISSUE #11: [Docker] Dockerize generator and importer

### PR DESCRIPTION
**Pull request by JonasCir**
@robbmue 
If you checkout the branch and run `docker-compose up -d` a sormas stack starts which then automatically gets populated with the test data. Then the importer shuts down and the main stack continues

---

Closes #11

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-      - autoheal=true
-      # override for stop timeout seconds during restart
-      - autoheal.stop.timeout=30
     links:
       - postgres
     image: hzibraunschweig/sormas-application:${SORMAS_DOCKER_VERSION}
+    ports:
+      - '6080:6080'
+      - '6081:6081'
+      - '6049:6049'
```
Comment (robbmue) [inline docker-compose.yml]: I think Payara is using 6048

File: docker-compose.yml [hunk 2]
```diff
@@ -90,6 +85,14 @@ services:
...
+    build:
+      context: src/
+      args:
+        - SORMAS_VERSION=${SORMAS_VERSION}
+        - OPENAPI_GENERATOR_VERSION=${OPENAPI_GENERATOR_VERSION}
+    environment:
+      - DB_HOST=${DB_HOST}
+    depends_on:
+      sormas:
+        condition: service_healthy
```
Comment (robbmue) [inline docker-compose.yml:98]: Very nice, works perfectly!

**Review by robbmue — APPROVED** [review]
Thank you so much for the fast fix! Everything's working now. 

Check the Payara admin port in docker-compose, it should be 6048.

File: src/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+WORKDIR /srv/
+COPY . /srv/
+
+
+ARG OPENAPI_GENERATOR_VERSION
+RUN ./importer/gen-client.sh
+COPY config/setup_and_run.sh setup_and_run.sh
+
+CMD ["./setup_and_run.sh"]
+
```
Comment (robbmue) [inline src/Dockerfile]: This should prevent new "doesn't work on my machine"-situations. 
Approved!

**Review by robbmue — DISMISSED** [review]
Nice work!

File: docker-compose.yml (hunk 1 above)
Comment (JonasCir) [reply docker-compose.yml]: Good catch fixed

File: docker-compose.yml (hunk 2 above)
Comment (JonasCir) [reply docker-compose.yml:98]: Thanks!

File: src/Dockerfile (hunk 3 above)
Comment (JonasCir) [reply src/Dockerfile]: Great :)

**Pull request merged**